<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculette Financi√®re</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <widget class="QWidget" name="tab_functions">
      <attribute name="title">
       <string>Fonctions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>VAN - Valeur Actuelle Nette d'un montant</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="LabelAnnuite">
              <property name="text">
               <string>&amp;Montant</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
              <property name="buddy">
               <cstring>LineEditAnnuite</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="LineEditAnnuite">
              <property name="maximumSize">
               <size>
                <width>85</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="LabelTauxActualisation">
              <property name="text">
               <string>&amp;Taux d'actualisation</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
              <property name="buddy">
               <cstring>DateEditDateActualisation</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="LineEditTauxActualisation">
              <property name="maximumSize">
               <size>
                <width>85</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string extracomment="fournir en pourcentage ex: 3.5"/>
              </property>
              <property name="whatsThis">
               <string extracomment="fournir en pourcentage ex: 3.5"/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="LabelDateActualisation">
              <property name="text">
               <string>&amp;Date d'actualisation</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
              <property name="buddy">
               <cstring>LineEditTauxActualisation</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDateEdit" name="DateEditDateActualisation">
              <property name="date">
               <date>
                <year>2014</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="displayFormat">
               <string>dd/MM/yyyy</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="LabelDateAnnuite">
              <property name="text">
               <string>Date du montant</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
              <property name="buddy">
               <cstring>DateEditDateAnnuite</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateEdit" name="DateEditDateAnnuite">
              <property name="date">
               <date>
                <year>2014</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="displayFormat">
               <string>dd/MM/yyyy</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonVanEval">
              <property name="text">
               <string>&amp;Calculer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditVanEval">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_obligation">
      <attribute name="title">
       <string>Obligation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="nominalLabel">
           <property name="text">
            <string>Nominal</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nominalLineEdit">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>102</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="tauxFacialLabel">
           <property name="text">
            <string>Taux Facial</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxTauxFacial">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="LabelTauxActuariel">
           <property name="text">
            <string>&amp;Taux Actuariel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
           <property name="buddy">
            <cstring>doubleSpinBoxTauxActuariel</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxTauxActuariel">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="dateEmissionLabel">
           <property name="text">
            <string>Date Emission</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateEdit" name="dateEditDateEmission">
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="dateDeJouissanceLabel">
           <property name="text">
            <string>Date de Jouissance</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDateEdit" name="dateEditDateJouissance">
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="dateDChAnceLabel">
           <property name="text">
            <string>Date d'echeance</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDateEdit" name="dateEditDateEcheance">
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="LabelDateEvaluation">
           <property name="text">
            <string>Date d'√©valuation</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDateEdit" name="dateEditDateEvaluation">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumDate">
            <date>
             <year>1799</year>
             <month>9</month>
             <day>14</day>
            </date>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::DaySection</enum>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QComboBox" name="comboBoxFrequenceCoupon">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>102</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxVisibleItems">
            <number>6</number>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QPushButton" name="pushButtonObligEval">
           <property name="text">
            <string>&amp;Calculer</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="evaluerLineEdit">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>102</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelSpread">
           <property name="text">
            <string>&amp;Spread</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
           <property name="buddy">
            <cstring>doubleSpinBoxSpread</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxSpread">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="labelFrequenceCoupon">
           <property name="text">
            <string>Fr√©quence de coupons</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_BOT">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>LineEditAnnuite</tabstop>
  <tabstop>LineEditTauxActualisation</tabstop>
  <tabstop>DateEditDateActualisation</tabstop>
  <tabstop>DateEditDateAnnuite</tabstop>
  <tabstop>pushButtonVanEval</tabstop>
  <tabstop>lineEditVanEval</tabstop>
  <tabstop>nominalLineEdit</tabstop>
  <tabstop>doubleSpinBoxTauxFacial</tabstop>
  <tabstop>doubleSpinBoxTauxActuariel</tabstop>
  <tabstop>doubleSpinBoxSpread</tabstop>
  <tabstop>dateEditDateEmission</tabstop>
  <tabstop>dateEditDateJouissance</tabstop>
  <tabstop>dateEditDateEcheance</tabstop>
  <tabstop>dateEditDateEvaluation</tabstop>
  <tabstop>comboBoxFrequenceCoupon</tabstop>
  <tabstop>pushButtonObligEval</tabstop>
  <tabstop>evaluerLineEdit</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
